package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "test": "npm run test -ws --if-present",
+    "format-all": "prettier --write .",
```

Comment:
Do we really need to add this here? We shouldn't need this anymore once all files are formatted. The tests will fail if someone opens a pull request with unformatted code.

Reply:
I would expect everyone to have Prettier running in the IDE when developing. If someone is working without it, it's their job to run Prettier somehow. They can do this with `npx prettier --write .` or however they like, but I think it shouldn't be our task to care about this use case. Developers working with this project, set up how it's meant to be, will never need this task, that's why I think we don't need it here.

Reply:
okay, if you prefer it like this.

Reply:
Only if it makes sense for you too 😄